src/Range.jsx

```diff
@@ -260,23 +260,25 @@ class Range extends React.Component {
     return true;
   }
 
-  trimAlignValue(v, nextProps = {}) {
+  trimAlignValue(v, i, nextProps = {}) {
     const mergedProps = { ...this.props, ...nextProps };
     const valInRange = utils.ensureValueInRange(v, mergedProps);
-    const valNotConflict = this.ensureValueNotConflict(valInRange, mergedProps);
+    const valNotConflict = this.ensureValueNotConflict(i, valInRange, mergedProps);
     return utils.ensureValuePrecision(valNotConflict, mergedProps);
   }
 
-  ensureValueNotConflict(val, { allowCross }) {
+  ensureValueNotConflict(i, val, { allowCross, pushable: thershold }) {
     const state = this.state || {};
-    const { handle, bounds } = state;
+    const { bounds } = state;
+    const handle = (i === undefined) ? state.handle : i;
```

Comment:
In which case `i` could be `undefiend`?

Reply:
yes, because `ensureValueNotConflict` invoked by `trimAlignValue`,  then `trimAlignValue` was invoked many place.
sometimes, only trim current handle value ( didn't pass index, like `calcValueByPos` in `createSlider.jsx` ) , but others, maybe map all handle values to trim.